ports/liburing/vcpkg.json

```
@@ -1,6 +1,6 @@
 {
   "name": "liburing",
-  "version": "2.9",
+  "version": "2.10",
   "description": "Linux-native io_uring I/O access library",
   "homepage": "https://github.com/axboe/liburing",
   "license": "(MIT OR LGPL-2.1) AND (MIT OR (GPL-2.0 WITH Linux-syscall-note))",
```

```suggestion
  "license": "(MIT OR LGPL-2.1) AND (MIT OR GPL-2.0 WITH Linux-syscall-note)",
```
Apply change of https://github.com/microsoft/vcpkg/pull/45673